<launch>

	<arg name="camera_info_topic"               default="/camera/left/camera_info" />
  <arg name="darknet_bounding_boxes_topics"   default="/darknet_ros/bounding_boxes" />
  <arg name="detection_class"                 default="person" />
  <arg name="queue_size"                      default="10" />

	<node name="object_localizer_node" pkg="object_localizer" type="object_localizer" output="screen">
        <param name="camera_info_topic" 	          value="$(arg camera_info_topic)"/>
        <param name="darknet_bounding_boxes_topic" 	value="$(arg darknet_bounding_boxes_topics)"/>
        <param name="detection_class" 	            value="$(arg detection_class)"/>
        <param name="queue_size" 	                  value="$(arg queue_size)"/>
 </node>

</launch>
